debian/focal/foreman-proxy/foreman-proxy.tmpfile

```diff
@@ -0,0 +1 @@
+d /run/foreman-proxy 0755 foreman-proxy foreman-proxy -
```

Comment:
This makes me wonder if we should ship this file in the repo itself so it's shared between RPMs and debs. Not for this patch, but this is a good time to review those things.